snap/snapcraft.yaml

```diff
@@ -18,6 +18,8 @@ environment:
 apps:
   rockcraft:
     command: bin/python $SNAP/bin/rockcraft
+  skopeo:
```

Comment:
I think this is a very good idea! My only possible concern is whether we then need to maintain this as part of the "public api" forever and ever
@sergiusens what do you think?

Reply:
ping @sergiusens